### ISSUE TITLE
Production Analytics API server reports status code 500

### ISSUE THREAD

**Issue by dobbz** [ekitabu-LLC/ekitabu-readium-desktop-working#69]
I'm still getting an HTML status code 500 on the Analytics API server.

```
> [2] Error: Request failed with status code 500
> [2]     at createError (C:\Projects\eKitabu\Readium\Desktop\ekitabu-readium-desktop-working-brettfork\node_modules\axios\lib\core\createError.js:16:15)
> [2]     at settle (C:\Projects\eKitabu\Readium\Desktop\ekitabu-readium-desktop-working-brettfork\node_modules\axios\lib\core\settle.js:18:12)
> [2]     at IncomingMessage.handleStreamEnd (C:\Projects\eKitabu\Readium\Desktop\ekitabu-readium-desktop-working-brettfork\node_modules\axios\lib\adapters\http.js:202:11)
> [2]     at IncomingMessage.emit (events.js:228:7)
> [2]     at endReadableNT (_stream_readable.js:1185:12)
> [2]     at processTicksAndRejections (internal/process/task_queues.js:81:21) {
> [2]   config: {
```

**Comment by mocyeg** [comment]
Resolved in #53

**Comment by dobbz** [comment]
I'm no longer seeing the 500 errors

Thanks @mocyeg
